src/encord_active/app/data_quality/sub_pages/explorer.py

```diff
@@ -160,14 +161,14 @@ def render_plotly_events(embedding_2d: DataFrame[Embedding2DSchema]) -> Optional
 
     new_selection = plotly_events(fig, click_event=False, select_event=True)
 
-    if new_selection != get_selection():
-        set_should_select(True)
-        set_selection(new_selection)
+    if new_selection != selection.value:
```

Comment:
So at this point, the `selection` is typed as a `List[dict]` but it's really `None` right?

Reply:
Perhaps we should type it `Optional[List[dict]]`

Reply:
Absolutely 👍